clang/test/OpenMP/target_map_messages.cpp

```diff
@@ -113,7 +114,7 @@ struct SA {
     #pragma omp target map(b[true:true])
     {}
 
-    #pragma omp target map(: c,f) // expected-error {{missing map type}}
+    #pragma omp target map(: c,f) // lt60-error {{missing map type}}
```

Comment:
I assume this also must be reported as an error for 6.0, no?

Reply:
In OMP60, the map-type can be omitted.  Will be default tofrom.

Reply:
So, does hanging `:` is allowed?

Reply:
Yes.

Reply:
Hm,, I see in 4.2 Clause Format that it is not allowed

Reply:
Hi Alexey, could you specify where 4.2 says that.  I back look at 6.8.3

map-type locator-list Keyword: alloc, delete,
from, release, to, tofrom
default

Thanks.

Reply:
Check page 100, the supported format is `[modifier-specification-list :]clause-argument-list`

Reply:
All those can be omitted, but error is needed for empty modifier-list?

Reply:
Yes, if you have dangling `:` without modifier list